[BUG] Error while starting AI since the switch to TOML

**Describe the bug**
Since the switch to TOML the AI produces an exception during the load.

**To Reproduce**
Run `pwnagotchi` without any modification.

**Expected behavior**
Do not produce any exception.

**Environment (please complete the following information):**
 - Pwnagotchi version: 52015014b41264db7cfd97719de14e612b93274c.
 - OS version: Raspbian.
 - Type of hardware: Raspberry Pi Zero W.

**Additional context**
Stack trace:
```
[2020-01-14 22:17:22,560] [ERROR] error while starting AI
Traceback (most recent call last):
  File "/usr/lib/pwnagotchi/lib/python3.7/site-packages/pwnagotchi/ai/__init__.py", line 45, in load
    a2c = A2C(MlpLstmPolicy, env, **config['params'])
  File "/usr/lib/pwnagotchi/lib/python3.7/site-packages/stable_baselines/a2c/a2c.py", line 86, in __init__
    self.setup_model()
  File "/usr/lib/pwnagotchi/lib/python3.7/site-packages/stable_baselines/a2c/a2c.py", line 118, in setup_model
    self.n_steps, n_batch_train, reuse=True, **self.policy_kwargs)
  File "/usr/lib/pwnagotchi/lib/python3.7/site-packages/stable_baselines/common/policies.py", line 682, in __init__
    layer_norm=False, feature_extraction="mlp", **_kwargs)
  File "/usr/lib/pwnagotchi/lib/python3.7/site-packages/stable_baselines/common/policies.py", line 425, in __init__
    input_sequence = batch_to_seq(extracted_features, self.n_env, n_steps)
  File "/usr/lib/pwnagotchi/lib/python3.7/site-packages/stable_baselines/a2c/utils.py", line 174, in batch_to_seq
    tensor_batch = tf.reshape(tensor_batch, [n_batch, n_steps, -1])
  File "/usr/lib/pwnagotchi/lib/python3.7/site-packages/tensorflow/python/ops/gen_array_ops.py", line 7179, in reshape
    "Reshape", tensor=tensor, shape=shape, name=name)
  File "/usr/lib/pwnagotchi/lib/python3.7/site-packages/tensorflow/python/framework/op_def_library.py", line 610, in _apply_op_helper
    param_name=input_name)
  File "/usr/lib/pwnagotchi/lib/python3.7/site-packages/tensorflow/python/framework/op_def_library.py", line 60, in _SatisfiesTypeConstraint
    ", ".join(dtypes.as_dtype(x).name for x in allowed_list)))
TypeError: Value passed to parameter 'shape' has DataType float32 not in list of allowed values: int32, int64
[2020-01-14 22:17:22,563] [WARNING] [ai] AI not loaded!
```

Hi, same problem Here, when updating the 1.4.3 to the toml switch.

Jan 15 10:13:15 pwnagotchi pwnagotchi-launcher[1389]: [2020-01-15 10:13:15,817] [ERROR] error while starting AI
Jan 15 10:13:15 pwnagotchi pwnagotchi-launcher[1389]: Traceback (most recent call last):
Jan 15 10:13:15 pwnagotchi pwnagotchi-launcher[1389]:   File "/usr/local/lib/python3.7/dist-packages/pwnagotchi/ai/__init__.py", line 45, in load
Jan 15 10:13:15 pwnagotchi pwnagotchi-launcher[1389]:     a2c = A2C(MlpLstmPolicy, env, **config['params'])
Jan 15 10:13:15 pwnagotchi pwnagotchi-launcher[1389]:   File "/usr/local/lib/python3.7/dist-packages/stable_baselines/a2c/a2c.py", line 86, in __init__
Jan 15 10:13:15 pwnagotchi pwnagotchi-launcher[1389]:     self.setup_model()
Jan 15 10:13:15 pwnagotchi pwnagotchi-launcher[1389]:   File "/usr/local/lib/python3.7/dist-packages/stable_baselines/a2c/a2c.py", line 118, in setup_model
Jan 15 10:13:15 pwnagotchi pwnagotchi-launcher[1389]:     self.n_steps, n_batch_train, reuse=True, **self.policy_kwargs)
Jan 15 10:13:15 pwnagotchi pwnagotchi-launcher[1389]:   File "/usr/local/lib/python3.7/dist-packages/stable_baselines/common/policies.py", line 682, in __init__
Jan 15 10:13:15 pwnagotchi pwnagotchi-launcher[1389]:     layer_norm=False, feature_extraction="mlp", **_kwargs)
Jan 15 10:13:15 pwnagotchi pwnagotchi-launcher[1389]:   File "/usr/local/lib/python3.7/dist-packages/stable_baselines/common/policies.py", line 425, in __init__
Jan 15 10:13:15 pwnagotchi pwnagotchi-launcher[1389]:     input_sequence = batch_to_seq(extracted_features, self.n_env, n_steps)
Jan 15 10:13:15 pwnagotchi pwnagotchi-launcher[1389]:   File "/usr/local/lib/python3.7/dist-packages/stable_baselines/a2c/utils.py", line 174, in batch_to_seq
Jan 15 10:13:15 pwnagotchi pwnagotchi-launcher[1389]:     tensor_batch = tf.reshape(tensor_batch, [n_batch, n_steps, -1])
Jan 15 10:13:15 pwnagotchi pwnagotchi-launcher[1389]:   File "/usr/lib/python3/dist-packages/tensorflow_core/python/ops/gen_array_ops.py", line 7714, in reshape
Jan 15 10:13:15 pwnagotchi pwnagotchi-launcher[1389]:     "Reshape", tensor=tensor, shape=shape, name=name)
Jan 15 10:13:15 pwnagotchi pwnagotchi-launcher[1389]:   File "/usr/lib/python3/dist-packages/tensorflow_core/python/framework/op_def_library.py", line 631, in _apply_op_helper
Jan 15 10:13:15 pwnagotchi pwnagotchi-launcher[1389]:     param_name=input_name)
Jan 15 10:13:15 pwnagotchi pwnagotchi-launcher[1389]:   File "/usr/lib/python3/dist-packages/tensorflow_core/python/framework/op_def_library.py", line 60, in _SatisfiesTypeConstraint
Jan 15 10:13:15 pwnagotchi pwnagotchi-launcher[1389]:     ", ".join(dtypes.as_dtype(x).name for x in allowed_list)))
Jan 15 10:13:15 pwnagotchi pwnagotchi-launcher[1389]: TypeError: Value passed to parameter 'shape' has DataType float32 not in list of allowed values: int32, int64
Jan 15 10:13:15 pwnagotchi pwnagotchi-launcher[1389]: [2020-01-15 10:13:15,909] [WARNING] [ai] AI not loaded!

I was on Version 1.4.3 (where the AI loaded) then did a git pull and pip3 install .

--- a/pwnagotchi/defaults.toml
+++ b/pwnagotchi/defaults.toml
@@ -4,7 +4,7 @@ main.custom_plugins = ""
 main.iface = "mon0"
 main.mon_start_cmd = "/usr/bin/monstart"
 main.mon_stop_cmd = "/usr/bin/monstop"
-main.mon_max_blind_epochs = 50.0
+main.mon_max_blind_epochs = 50
 main.no_restart = false
 main.whitelist = [
   "EXAMPLE_NETWORK",
@@ -23,13 +23,13 @@ main.plugins.grid.exclude = [
 
 main.plugins.auto-update.enabled = true
 main.plugins.auto-update.install = true
-main.plugins.auto-update.interval = 1.0
+main.plugins.auto-update.interval = 1
 
 main.plugins.net-pos.enabled = false
 main.plugins.net-pos.api_key = "test"
 
 main.plugins.gps.enabled = false
-main.plugins.gps.speed = 19200.0
+main.plugins.gps.speed = 19200
 main.plugins.gps.device = "/dev/ttyUSB0"
 
 main.plugins.webgpsmap.enabled = false
@@ -48,26 +48,26 @@ main.plugins.wigle.api_key = ""
 main.plugins.bt-tether.enabled = false
 
 main.plugins.bt-tether.devices.android-phone.enabled = false
-main.plugins.bt-tether.devices.android-phone.search_order = 1.0
+main.plugins.bt-tether.devices.android-phone.search_order = 1
 main.plugins.bt-tether.devices.android-phone.mac = ""
 main.plugins.bt-tether.devices.android-phone.ip = "192.168.44.44"
-main.plugins.bt-tether.devices.android-phone.netmask = 24.0
-main.plugins.bt-tether.devices.android-phone.interval = 1.0
-main.plugins.bt-tether.devices.android-phone.scantime = 10.0
-main.plugins.bt-tether.devices.android-phone.max_tries = 10.0
+main.plugins.bt-tether.devices.android-phone.netmask = 24
+main.plugins.bt-tether.devices.android-phone.interval = 1
+main.plugins.bt-tether.devices.android-phone.scantime = 10
+main.plugins.bt-tether.devices.android-phone.max_tries = 10
 main.plugins.bt-tether.devices.android-phone.share_internet = false
-main.plugins.bt-tether.devices.android-phone.priority = 1.0
+main.plugins.bt-tether.devices.android-phone.priority = 1
 
 main.plugins.bt-tether.devices.ios-phone.enabled = false
-main.plugins.bt-tether.devices.ios-phone.search_order = 2.0
+main.plugins.bt-tether.devices.ios-phone.search_order = 2
 main.plugins.bt-tether.devices.ios-phone.mac = ""
 main.plugins.bt-tether.devices.ios-phone.ip = "172.20.10.6"
-main.plugins.bt-tether.devices.ios-phone.netmask = 24.0
-main.plugins.bt-tether.devices.ios-phone.interval = 5.0
-main.plugins.bt-tether.devices.ios-phone.scantime = 20.0
-main.plugins.bt-tether.devices.ios-phone.max_tries = 0.0
+main.plugins.bt-tether.devices.ios-phone.netmask = 24
+main.plugins.bt-tether.devices.ios-phone.interval = 5
+main.plugins.bt-tether.devices.ios-phone.scantime = 20
+main.plugins.bt-tether.devices.ios-phone.max_tries = 0
 main.plugins.bt-tether.devices.ios-phone.share_internet = false
-main.plugins.bt-tether.devices.ios-phone.priority = 999.0
+main.plugins.bt-tether.devices.ios-phone.priority = 999
 
 main.plugins.memtemp.enabled = false
 main.plugins.memtemp.scale = "celsius"
@@ -79,8 +79,8 @@ main.plugins.paw-gps.ip = ""
 main.plugins.gpio_buttons.enabled = false
 
 main.plugins.led.enabled = true
-main.plugins.led.led = 0.0
-main.plugins.led.delay = 200.0
+main.plugins.led.led = 0
+main.plugins.led.delay = 200
 main.plugins.led.patterns.loaded = "oo  oo  oo oo  oo  oo  oo"
 main.plugins.led.patterns.updating = "oo  oo  oo oo  oo  oo  oo"
 main.plugins.led.patterns.unread_inbox = "oo  oo  oo oo  oo  oo  oo"
@@ -113,37 +113,37 @@ main.log.rotation.size = "10M"
 ai.enabled = true
 ai.path = "/root/brain.nn"
 ai.laziness = 0.1
-ai.epochs_per_episode = 50.0
+ai.epochs_per_episode = 50
 
 ai.params.gamma = 0.99
-ai.params.n_steps = 1.0
+ai.params.n_steps = 1
 ai.params.vf_coef = 0.25
 ai.params.ent_coef = 0.01
 ai.params.max_grad_norm = 0.5
 ai.params.learning_rate = 0.001
 ai.params.alpha = 0.99
 ai.params.epsilon = 0.00001
-ai.params.verbose = 1.0
+ai.params.verbose = 1
 ai.params.lr_schedule = "constant"
 
 personality.advertise = true
 personality.deauth = true
 personality.associate = true
 personality.channels = []
-personality.min_rssi = -200.0
-personality.ap_ttl = 120.0
-personality.sta_ttl = 300.0
-personality.recon_time = 30.0
-personality.max_inactive_scale = 2.0
-personality.recon_inactive_multiplier = 2.0
-personality.hop_recon_time = 10.0
-personality.min_recon_time = 5.0
-personality.max_interactions = 3.0
-personality.max_misses_for_recon = 5.0
-personality.excited_num_epochs = 10.0
-personality.bored_num_epochs = 15.0
-personality.sad_num_epochs = 25.0
-personality.bond_encounters_factor = 20000.0
+personality.min_rssi = -200
+personality.ap_ttl = 120
+personality.sta_ttl = 300
+personality.recon_time = 30
+personality.max_inactive_scale = 2
+personality.recon_inactive_multiplier = 2
+personality.hop_recon_time = 10
+personality.min_recon_time = 5
+personality.max_interactions = 3
+personality.max_misses_for_recon = 5
+personality.excited_num_epochs = 10
+personality.bored_num_epochs = 15
+personality.sad_num_epochs = 25
+personality.bond_encounters_factor = 20000
 
 ui.fps = 0.0
 
@@ -175,17 +175,17 @@ ui.web.address = "0.0.0.0"
 ui.web.username = "changeme"
 ui.web.password = "changeme"
 ui.web.origin = ""
-ui.web.port = 8080.0
+ui.web.port = 8080
 ui.web.on_frame = ""
 
 ui.display.enabled = true
-ui.display.rotation = 180.0
+ui.display.rotation = 180
 ui.display.type = "waveshare_2"
 ui.display.color = "black"
 
 bettercap.scheme = "http"
 bettercap.hostname = "localhost"
-bettercap.port = 8081.0
+bettercap.port = 8081
 bettercap.username = "pwnagotchi"
 bettercap.password = "pwnagotchi"
 bettercap.handshakes = "/root/handshakes"

